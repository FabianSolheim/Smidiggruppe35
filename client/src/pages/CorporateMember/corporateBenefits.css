.joinUsBtnBenefits {
    font-family: "Montserrat", sans-serif;
    border-radius: 0;
    height: 50px;
    width: 200px;
    background-color: #034F7A;
    color: white;
    border: 0;
    font-size: 16px;
    display:block; margin: 0 auto
}
.vettedNpoImg {
    margin-left: 260px;
}
.dashboardImg {
    max-width: 462px;
    max-height: 280px;
    object-fit: contain;
}
.joinUsDivCorporate {
    margin-top: -170px;
}
@media only screen and (max-width: 900px) {

    .dashboardImg {
        width: 300px;
        margin-right: auto;
        margin-top: 40px;

    }
    .vettedNpoImg {
        width: 150px;
        margin-left: auto;
    }
    .textCorporateBenefits {
        padding: 20px;
    }
    .joinUsDivCorporate {
        margin-top: 40px;
        display: inherit;
    }

}