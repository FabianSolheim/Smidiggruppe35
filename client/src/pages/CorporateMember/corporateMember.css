.corporateHeader {
    width: 100%;
    height: 700px;
    background-image: url("https://images.unsplash.com/photo-1509099863731-ef4bff19e808?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2072&q=80");
    background-repeat: no-repeat;
    background-size: cover;
    background-position: center;
    text-align: center;
    font-size: 64px;
}
.corporateHeader em {
    color: #034F7A;
    font-style: normal;
}