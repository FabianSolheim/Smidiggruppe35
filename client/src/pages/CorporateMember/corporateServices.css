.containerCorporateServices {
    background-color: rgba(3,79,122,0.1);
    width: 100%;
    height: 700px;
    display: flex;
    align-items: center;
}
.widthContainerServices {
    width: 1440px;
    margin: auto;
}