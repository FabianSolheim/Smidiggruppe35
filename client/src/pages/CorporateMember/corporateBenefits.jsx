import React from "react";
import Box from "@mui/material/Box";
import Grid from "@mui/material/Grid";
import "./corporateBenefits.css";
import { HeadlineWithUnderline } from "../../components/HeadlineWithUnderline/headlineWithUnderline";
import { BtnWithHeadline } from "../../components/BtnWithHeadline/BtnWithHeadline";
import { Container } from "@mui/material";

export function CorporateBenefits() {
  return (
    <>
      {HeadlineWithUnderline("Benefits", 100, "#034F7A", 50)}

      <Box
        sx={{ flexGrow: 1 }}
        style={{
          maxWidth: 1240,
          margin: "auto",
        }}
      >
        <Grid container spacing={0}>
          <Grid item xs={12} sm={6}>
            <Container>
              <p className="textCorporateBenefits">
                By using Meliora Impact's platform, companies can leverage their
                corporate resources for greater social good.
                <br />
                <br />
                Donors who choose our platform get access to carefully selected
                nonprofits and projects to support, will receive inside stories
                about the projects and will see how their donations make an
                impact.
                <br />
                <br />
                Donors get access to engaging reports on their philanthropic
                activity, continuously updated dashboards, and of course have
                full control over their donations.
                <br />
                <br />
                In addition, corporate donors can sign up to receive a
                philanthropic strategy fully tailored to their business and
                values.
              </p>
            </Container>
          </Grid>
          <Grid item xs={12} sm={6}>
            <Container>
              <div>
                <img
                  className="dashboardImg"
                  style={{
                    display: "block",
                    marginLeft: "auto",
                  }}
                  src={require("../../images/InsightDesktop.svg")}
                />
              </div>
              <div style={{ marginTop: 30 }}>
                <img
                  className="vettedNpoImg"
                  style={{
                    display: "block",
                    marginRight: "auto",
                  }}
                  src={require("../../images/20vettednpo.svg")}
                />
              </div>
            </Container>
          </Grid>
          <Grid item xs={12} md={6}>
            <div className={"joinUsDivCorporate"}>
              {BtnWithHeadline("Ready to get Started?", "Join us")}
            </div>
          </Grid>
        </Grid>
      </Box>
    </>
  );
}
