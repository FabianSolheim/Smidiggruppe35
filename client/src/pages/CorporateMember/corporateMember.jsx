import React from "react";
import "./corporateMember.css";
import { CorporateBenefits } from "./corporateBenefits";
import { CorporateServices } from "./corporateServices";
import { HeaderBox } from "../../components/HeaderBox/HeaderBox";

export function CorporateMember() {
  return (
    <div>
      {HeaderBox(
        "https://images.unsplash.com/photo-1509099836639-18ba1795216d?crop=entropy&cs=tinysrgb&fm=jpg&ixlib=rb-1.2.1&q=80&raw_url=true&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=3862",
        <div>
          Corporate <em>members</em>
        </div>,
        "white",
        <div>
          Joining Meloria Impact gives you access to a{" "}
          <strong>
            <em>broad variety</em>
          </strong>{" "}
          of{" "}
          <strong>
            <em>helpful tools</em>
          </strong>{" "}
          for tracking, statistics & more
        </div>,
        "white"
      )}
      <div style={{ height: 60 }} />
      <CorporateBenefits />
      <div style={{ height: 60 }} />
      <CorporateServices />
      <div style={{ height: 100 }} />
    </div>
  );
}
