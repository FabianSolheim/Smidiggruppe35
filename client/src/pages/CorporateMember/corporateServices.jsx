import React from "react";
import Box from "@mui/material/Box";
import Grid from "@mui/material/Grid";
import "./corporateServices.css";
import { Underline } from "../../components/Underline/underline";

export function CorporateServices() {
  return (
    <>
      <div>
        <h1
          style={{
            fontSize: 36,
            textAlign: "center",
          }}
        >
          Services
        </h1>
        {Underline(100, "#034F7A")}
      </div>
      <div className="containerCorporateServices">
        <div className="widthContainerServices">
          <Box sx={{ flexGrow: 1 }}>
            <Grid container spacing={2}>
              <Grid item xs={12} md={4}>
                <div style={{ textAlign: "center" }}>
                  <img src={require("../../images/currentdonation.svg")} />
                  <p
                    style={{
                      fontSize: 18,
                      color: "#034F7A",
                      marginTop: 10,
                    }}
                  >
                    <strong>Curated Donations</strong>
                  </p>
                  <p
                    style={{
                      fontSize: 16,
                      marginTop: 8,
                    }}
                  >
                    <strong>Regular</strong> funding
                    <br /> stream
                  </p>
                </div>
              </Grid>
              <Grid item xs={12} md={4}>
                <div style={{ textAlign: "center", margin: 20 }}>
                  <img src={require("../../images/unabridgeddonations.svg")} />
                  <p
                    style={{
                      fontSize: 18,
                      color: "#034F7A",
                      marginTop: 10,
                    }}
                  >
                    <strong>Platform</strong>
                  </p>
                  <p
                    style={{
                      fontSize: 16,
                      marginTop: 8,
                    }}
                  >
                    One platform <strong>connecting</strong>
                    <br />
                    them to their
                    <br />
                    <strong>donors</strong>
                  </p>
                </div>
              </Grid>
              <Grid item xs={12} md={4}>
                <div style={{ textAlign: "center", margin: 20 }}>
                  <img src={require("../../images/inclusivegiving.svg")} />
                  <p
                    style={{
                      fontSize: 18,
                      color: "#034F7A",
                      marginTop: 10,
                    }}
                  >
                    <strong>Tools</strong>
                  </p>
                  <p
                    style={{
                      fontSize: 16,
                      marginTop: 8,
                    }}
                  >
                    <strong>Free</strong> digital
                    <br /> tools
                  </p>
                </div>
              </Grid>
              <Grid item xs={12} md={4}>
                <div style={{ textAlign: "center", marginTop: 10 }}>
                  <img src={require("../../images/scheduledgiving.svg")} />
                  <p
                    style={{
                      fontSize: 18,
                      color: "#034F7A",
                      marginTop: 10,
                    }}
                  >
                    <strong>Funding</strong>
                  </p>
                  <p
                    style={{
                      fontSize: 16,
                      marginTop: 8,
                    }}
                  >
                    <strong>Regular</strong> funding
                    <br /> stream
                  </p>
                </div>
              </Grid>
              <Grid item xs={12} md={4}>
                <div style={{ textAlign: "center", margin: 20 }}>
                  <img src={require("../../images/usefulinsights.svg")} />
                  <p
                    style={{
                      fontSize: 18,
                      color: "#034F7A",
                      marginTop: 10,
                    }}
                  >
                    <strong>Platform</strong>
                  </p>
                  <p
                    style={{
                      fontSize: 16,
                      marginTop: 8,
                    }}
                  >
                    One platform <strong>connecting</strong>
                    <br />
                    them to their
                    <br />
                    <strong>donors</strong>
                  </p>
                </div>
              </Grid>
              <Grid item xs={12} md={4}>
                <div style={{ textAlign: "center", margin: 20 }}>
                  <img src={require("../../images/searchfilter.svg")} />
                  <p
                    style={{
                      fontSize: 18,
                      color: "#034F7A",
                      marginTop: 10,
                    }}
                  >
                    <strong>Tools</strong>
                  </p>
                  <p
                    style={{
                      fontSize: 16,
                      marginTop: 8,
                    }}
                  >
                    <strong>Free</strong> digital
                    <br /> tools
                  </p>
                </div>
              </Grid>
              <Grid item xs={12} md={4}>
                <div style={{ textAlign: "center", margin: 20 }}>
                  <img src={require("../../images/regularreports.svg")} />
                  <p
                    style={{
                      fontSize: 18,
                      color: "#034F7A",
                      marginTop: 10,
                    }}
                  >
                    <strong>Tools</strong>
                  </p>
                  <p
                    style={{
                      fontSize: 16,
                      marginTop: 8,
                    }}
                  >
                    <strong>Free</strong> digital
                    <br /> tools
                  </p>
                </div>
              </Grid>
              <Grid item xs={12} md={4}>
                <div style={{ textAlign: "center", margin: 20 }}>
                  <img src={require("../../images/insidestories.svg")} />
                  <p
                    style={{
                      fontSize: 18,
                      color: "#034F7A",
                      marginTop: 10,
                    }}
                  >
                    <strong>Tools</strong>
                  </p>
                  <p
                    style={{
                      fontSize: 16,
                      marginTop: 8,
                    }}
                  >
                    <strong>Free</strong> digital
                    <br /> tools
                  </p>
                </div>
              </Grid>
              <Grid item xs={12} md={4}>
                <div style={{ textAlign: "center", margin: 20 }}>
                  <img src={require("../../images/bookconsultations.svg")} />
                  <p
                    style={{
                      fontSize: 18,
                      color: "#034F7A",
                      marginTop: 10,
                    }}
                  >
                    <strong>Tools</strong>
                  </p>
                  <p
                    style={{
                      fontSize: 16,
                      marginTop: 8,
                    }}
                  >
                    <strong>Free</strong> digital
                    <br /> tools
                  </p>
                </div>
              </Grid>
            </Grid>
          </Box>
        </div>
      </div>
    </>
  );
}
