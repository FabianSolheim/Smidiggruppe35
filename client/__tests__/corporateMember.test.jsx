import React from "react";
import ReactDOM from "react-dom";
import { CorporateMember } from "../src/pages/CorporateMember/corporateMember";

describe("CorporateMember page", () => {
  it("snapshot corporateMember page", () => {
    const domElement = document.createElement("div");
    ReactDOM.render(<CorporateMember />, domElement);
    expect(domElement.innerHTML).toMatchSnapshot();
  });
});
