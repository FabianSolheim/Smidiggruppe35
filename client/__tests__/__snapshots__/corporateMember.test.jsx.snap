// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CorporateMember page snapshot corporateMember page 1`] = `"<div><div class=\\"TopHeader\\" style=\\"background-image: url(https://images.unsplash.com/photo-1509099836639-18ba1795216d?crop=entropy&amp;cs=tinysrgb&amp;fm=jpg&amp;ixlib=rb-1.2.1&amp;q=80&amp;raw_url=true&amp;ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&amp;auto=format&amp;fit=crop&amp;w=3862); background-repeat: no-repeat; background-size: cover; background-position: center; text-align: center;\\"><div class=\\"HeaderBox\\"><div class=\\"BoxHeadline\\" style=\\"color: white;\\"><h1>Corporate <em>members</em></h1></div><div class=\\"BoxText\\" style=\\"color: white;\\"><p>Joining Meloria Impact gives you access to a <strong><em>broad variety</em></strong><br> of <strong><em>helpful tools</em></strong> for tracking, statistics &amp; more</p></div></div></div><div style=\\"height: 100px;\\"></div><div><h1 style=\\"font-size: 36px; font-weight: bold; text-align: center;\\">Benefits</h1></div><div style=\\"width: 100px; height: 2px; background-color: rgb(3, 79, 122); margin: 5px auto 100px auto;\\"></div><div class=\\"MuiBox-root css-i9gxme\\" style=\\"max-width: 1240px; margin: auto;\\"><div class=\\"MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root\\"><div class=\\"MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-6 css-1fyyp8j-MuiGrid-root\\"><p class=\\"textCorporateBenefits\\">By using Meliora Impact's platform, companies can leverage their corporate resources for greater social good.<br><br>Donors who choose our platform get access to carefully selected nonprofits and projects to support, will receive inside stories about the projects and will see how their donations make an impact.<br><br>Donors get access to engaging reports on their philanthropic activity, continuously updated dashboards, and of course have full control over their donations.<br><br>In addition, corporate donors can sign up to receive a philanthropic strategy fully tailored to their business and values.</p><h1 style=\\"margin-top: 80px; margin-bottom: 20px; font-size: 30px; text-align: center;\\">Ready to get started?</h1><button class=\\"joinUsBtnBenefits\\">Join us</button></div><div class=\\"MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-6 css-1fyyp8j-MuiGrid-root\\"><div><img class=\\"dashboardImg\\" style=\\"display: block; margin-left: auto;\\" src=\\"test-file-stub\\"></div><div style=\\"margin-top: 30px;\\"><img class=\\"vettedNpoImg\\" style=\\"display: block; margin-right: auto;\\" src=\\"test-file-stub\\"></div></div></div></div><div style=\\"height: 100px;\\"></div><div><h1 style=\\"font-size: 36px; font-weight: bold; text-align: center;\\">Services</h1></div><div style=\\"width: 100px; height: 2px; background-color: rgb(3, 79, 122); margin: 5px auto 50px auto;\\"></div><div class=\\"containerCorporateServices\\"><div class=\\"widthContainerServices\\"><div class=\\"MuiBox-root css-i9gxme\\"><div class=\\"MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 css-mhc70k-MuiGrid-root\\"><div class=\\"MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 css-1lj5egr-MuiGrid-root\\"><div style=\\"text-align: center;\\"><img src=\\"test-file-stub\\"><p style=\\"font-size: 18px; color: rgb(3, 79, 122); margin-top: 10px;\\"><strong>Curated Donations</strong></p><p style=\\"font-size: 16px; margin-top: 8px;\\"><strong>Regular</strong> funding<br> stream</p></div></div><div class=\\"MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 css-1lj5egr-MuiGrid-root\\"><div style=\\"text-align: center; margin: 20px;\\"><img src=\\"test-file-stub\\"><p style=\\"font-size: 18px; color: rgb(3, 79, 122); margin-top: 10px;\\"><strong>Platform</strong></p><p style=\\"font-size: 16px; margin-top: 8px;\\">One platform <strong>connecting</strong><br>them to their<br><strong>donors</strong></p></div></div><div class=\\"MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 css-1lj5egr-MuiGrid-root\\"><div style=\\"text-align: center; margin: 20px;\\"><img src=\\"test-file-stub\\"><p style=\\"font-size: 18px; color: rgb(3, 79, 122); margin-top: 10px;\\"><strong>Tools</strong></p><p style=\\"font-size: 16px; margin-top: 8px;\\"><strong>Free</strong> digital<br> tools</p></div></div><div class=\\"MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 css-1lj5egr-MuiGrid-root\\"><div style=\\"text-align: center; margin-top: 10px;\\"><img src=\\"test-file-stub\\"><p style=\\"font-size: 18px; color: rgb(3, 79, 122); margin-top: 10px;\\"><strong>Funding</strong></p><p style=\\"font-size: 16px; margin-top: 8px;\\"><strong>Regular</strong> funding<br> stream</p></div></div><div class=\\"MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 css-1lj5egr-MuiGrid-root\\"><div style=\\"text-align: center; margin: 20px;\\"><img src=\\"test-file-stub\\"><p style=\\"font-size: 18px; color: rgb(3, 79, 122); margin-top: 10px;\\"><strong>Platform</strong></p><p style=\\"font-size: 16px; margin-top: 8px;\\">One platform <strong>connecting</strong><br>them to their<br><strong>donors</strong></p></div></div><div class=\\"MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 css-1lj5egr-MuiGrid-root\\"><div style=\\"text-align: center; margin: 20px;\\"><img src=\\"test-file-stub\\"><p style=\\"font-size: 18px; color: rgb(3, 79, 122); margin-top: 10px;\\"><strong>Tools</strong></p><p style=\\"font-size: 16px; margin-top: 8px;\\"><strong>Free</strong> digital<br> tools</p></div></div><div class=\\"MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 css-1lj5egr-MuiGrid-root\\"><div style=\\"text-align: center; margin: 20px;\\"><img src=\\"test-file-stub\\"><p style=\\"font-size: 18px; color: rgb(3, 79, 122); margin-top: 10px;\\"><strong>Tools</strong></p><p style=\\"font-size: 16px; margin-top: 8px;\\"><strong>Free</strong> digital<br> tools</p></div></div><div class=\\"MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 css-1lj5egr-MuiGrid-root\\"><div style=\\"text-align: center; margin: 20px;\\"><img src=\\"test-file-stub\\"><p style=\\"font-size: 18px; color: rgb(3, 79, 122); margin-top: 10px;\\"><strong>Tools</strong></p><p style=\\"font-size: 16px; margin-top: 8px;\\"><strong>Free</strong> digital<br> tools</p></div></div><div class=\\"MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 css-1lj5egr-MuiGrid-root\\"><div style=\\"text-align: center; margin: 20px;\\"><img src=\\"test-file-stub\\"><p style=\\"font-size: 18px; color: rgb(3, 79, 122); margin-top: 10px;\\"><strong>Tools</strong></p><p style=\\"font-size: 16px; margin-top: 8px;\\"><strong>Free</strong> digital<br> tools</p></div></div></div></div></div></div><div style=\\"height: 100px;\\"></div></div>"`;
